utils/regression-analysis/src/weights.rs

```diff
@@ -0,0 +1,688 @@
+// This file is part of Gear.
+
+// Copyright (C) 2022 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Autogenerated trait implementation for regression analysis tool
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-08-10
+//! EXECUTION: Some(Native), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// C:\Users\ark0f\Projects\gear\target\release\gear-node.exe benchmark pallet --steps 1 --repeat 1 --chain=dev --pallet=pallet_gear --extrinsic=* --execution=native --heap-pages=4096 --output .\utils\regression-analysis\src\weights.rs --template .\.maintain\regression-analysis-weight-template.rs.hbs
```

Comment:
Whether this file is must be in our repo? We have already one weights.rs, can we use it for comparing?

Reply:
We cannot because there are hardcoded weights. While we need dynamically update them during each `test-measurements.yml` run